src/test/java/tests/ClickAccount.java

```diff
@@ -1,19 +1,26 @@
 package tests;
 
+import org.testng.annotations.Test;
+import pages.Account;
 import pages.LoginPage;
 import pages.ProfilePage;
 import utils.Driver;
 
 import java.time.Duration;
 
-public class ProfileAndAccount {
+public class ClickAccount extends TestBase {
 
+    @Test
     public void ProfileAndAccount() {
 
       new LoginTests().positiveLogin();
-        ProfilePage profilePage = new ProfilePage();
-        ProfilePage
         Driver.getDriver().manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-        profilePage.getAccountLink().click();
```

Comment:
this one too

Reply:
Thanks for feedback, i will send you corrected ones